TASVideos/Pages/UserFiles/Upload.cshtml.cs

```diff
@@ -86,6 +86,16 @@ public async Task<IActionResult> OnPost()
 			return Page();
 		}
 
+		byte[] actualFileData;
+
+		{
```

Comment:
These curlies seem unnecessary, also stylecop nags

Reply:
`tempStream` and `gzip` only have meaning within this block; the local scope keeps them from confusing the rest of the method.  I don't care about stylecop; any formatting nag that doesn't just automatically do the formatting for you is a complete waste of developer time.